hack/setup-build-index-image.sh

```diff
@@ -50,7 +53,7 @@ spec:
         - |
           set -xe
 
-          yum install jq git wget -y
+          yum install jq git wget make buildah go yq -y
```

Comment:
I think we can remove this new packages right?
I don't see where we use them

Reply:
Correct! That's a leftover